portfolio/src/main/webapp/script.js

```diff
@@ -43,6 +43,7 @@ function loadHTML(src, selector, position) {
     .then(response => response.text())
     .then(text => {
       document.querySelector(selector)?.insertAdjacentHTML(position, text);
+      console.log(`Loaded ${src} ${position}ing of ${selector}`);
```

Comment:
I assume you don't need this.

Reply:
Correct, I was using this for debugging.
Funnily enough, before finding out I needed to disable caching, a print elsewhere actually made the loading more consistent. This makes me think it's a concurrency issue.
Should I remove this line?

Reply:
Yeah typically you try to keep debugging logic out of your master branch

Reply:
Gotcha, I'll keep that in mind. Thanks :)